RxNimble.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "RxNimble"
-  s.version      = "5.0.0"
+  s.version      = "6.0.0"
```

Comment:
This is actually the version number for `RxNimble`, I'll change it to 5.1.0 in our release.

Reply:
I guessed 6 since it seemed a lot of the other community repos were updating the major version number

Reply:
Yeah, there's something to be said about staying in lock-step with RxSwift, but this way we can make our own breaking changes independent of them.